css/properties/grid-template-rows.json

```diff
@@ -466,66 +304,34 @@
                   "version_removed": "57",
                   "notes": "<a href='https://developer.mozilla.org/docs/Web/CSS/calc'><code>calc()</code></a> doesn't work in <code>repeat()</code> (see <a href='https://bugzil.la/1350069'>bug 1350069</a>).",
                   "partial_implementation": true
-                },
-                {
-                  "version_added": "40",
-                  "version_removed": "59",
-                  "flags": [
-                    {
-                      "type": "preference",
-                      "name": "layout.css.grid.enabled",
-                      "value_to_set": "true"
-                    }
-                  ]
                 }
               ],
               "firefox_android": [
                 {
-                  "version_added": "52"
+                  "version_added": "76"
```

Comment:
Here's the other irregular change to look extra carefully at.

Reply:
Looks like the same case here, so we're all good!